nixos/modules/tasks/cpu-freq.nix

```diff
@@ -4,22 +4,43 @@ with lib;
 
 let
   cpupower = config.boot.kernelPackages.cpupower;
-  cfg = config.powerManagement;
+  cfg = config.powerManagement.cpufreq;
 in
 
 {
   ###### interface
 
-  options = {
+  options.powerManagement.cpufreq = {
 
-    powerManagement.cpuFreqGovernor = mkOption {
+    governor = mkOption {
```

Comment:
I don't think we should break the configuration within a release also it has an alias without a good reason.

Reply:
Adding new options should be fine.add games/frogatto

Reply:
@Mic92 The alias allow users on 18.09 to upgrade to the new module name (`powerManagement.cpufreq.governor`) which will be used in 19.03 by default.

I don't think there is a good reason *not* to set this.  I don't think adding an alias will break anyone's configuration?